### PR TITLE
Update Tips'n'Tricks

### PR DESCRIPTION
**Pull request by thom-at-redhat**
Added File Paths section

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 667068523fae50b1b0bb915ccad0656577c3ce51:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: docs/guides/tipsntricks.rst [hunk 1]
```diff
@@ -114,3 +114,22 @@ Test generation (testgen)
...
+For relative file pathnames in the project, there are several helper functions that can be used:
+
+* utils.path.project_path for project root (e.g. cfme_tests)
+* utils.path.conf_path for configuration directory (e.g. cfme_tests/conf)
+* utils.path.data_path for data directory (e.g. cfme_tests/data)
+* utils.path.docs_path for documentation directory (e.g. cfme_tests/docs)
+* utils.path.log_path for logfile directory (e.g. cfme_tests/log)
+* utils.path.scripts_path for scripts directory (e.g. cfme_tests/scripts)
+* utils.path.scripts_data_path for scripts data directory (e.g. cfme_tests/scripts/data)
+* utils.path.template_path for template directory (e.g. cfme_tests/template)
```
Comment (seandst) [inline docs/guides/tipsntricks.rst]: I don't want to be on the hook to maintain the list, and I think the autogenerated one on the `utils.path` module page is good enough (or could be made good enough if the docstrings in the module need improvement).

`utils.path` works for all paths (including absolute ones) relative to the project root, not just relative file paths in the project root. (i.e. change "For relative file pathnames in the project root" to "For any path in the project root").

**Comment by dajoRH** [comment]
Lint report for commit 698b6922b39ec939ed08d0bbfd7b0ebea5ea301d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by seandst** [comment]
I merged it with some tweaks: https://github.com/RedHatQE/cfme_tests/commit/5ab124e51300135fa29022a2686d2c74d9ff2932

The main thing was to switch it to use the sphinx link instead of a hard-coded one, so it'll work anywhere that it's built.

I also removed the reference to get_rel_path under similar objections as before; I'd rather let individual module docs speak for themselves than have to maintain references.

If we want to provide examples for usage, they should be where they do the most good, but I think they already are, since the place they do the most good is (in my opinion) the `py.path.local` docs. However, if we want examples that are specific to this project, I think they belond directly in `utils.path` in a generic way, since we don't know what the future of utils.ssh holds. In reviewing this, I've seen a few places where my failure to do this has led to stale docs unnecessarily. Sorry 'bout that. :(

Finally, I removed the bit about combining imports, since that belongs (and [is](https://www.python.org/dev/peps/pep-0008/#imports)) in the style guide.

**Pull request merged**